static/schemas/neon-api/v1/neon_gasPrice/response.json

```diff
@@ -0,0 +1,141 @@
+{
+  "type": "object",
+  "description": "Response to request for transaction gas price",
+  "required": [
+    "jsonrpc",
+    "id",
+    "result"
+  ],
+  "properties": {
+    "jsonrpc": {
+      "type": "string",
+      "default": "2.0",
+      "description": "The JSON RPC version"
+    },
+    "id": {
+      "type": "string",
+      "examples": [
+        "OpEgqw7r98qwerqwehkjhwe8"
+      ],
+      "description": "End-user provided identifier for the request"
+    },
+    "result": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "gas_price": {
+            "description": "Gas fee as hexadecimal value in Galan (NEON).",
+            "type": "string",
+            "format": "hexadecimal",
+            "examples": [
+              "0x0"
+            ]
+          },
+          "suggested_gas_price": {
+            "description": "Suggested gas price as hexadecimal value in Galan (NEON).",
+            "type": "string",
+            "format": "hexadecimal",
+            "examples": [
+              "0x10c2c383c6"
+            ]
+          },
+          "min_acceptable_gas_price": {
+            "description": "Minimum acceptable gas price as hexadecimal value in Galan (NEON).",
+            "type": "string",
+            "format": "hexadecimal",
+            "examples": [
+              "0x3b9aca00"
+            ]
+          },
+          "min_executable_gas_price": {
+            "description": "Minimum executable gas price as hexadecimal value in Galan (NEON).",
```

Comment:
ping

Reply:
Minimal gas-price in tx for execution by the operator.
min_executable_gas_price can go up and go down because it depends on the SOL price and NEON price.

tx is executed if gas-price is bigger than min_executable_gas_price

Reply:
ditto -- is this set by Neon, by the Operator -- is it hard-coded or a variable?